code/AssetLib/Ogre/OgreXmlSerializer.cpp

```diff
@@ -490,7 +490,7 @@ bool OgreXmlSerializer::ImportSkeleton(Assimp::IOSystem *pIOHandler, MeshXml *me
     OgreXmlSerializer serializer(xmlParser.get());
     XmlNode root = xmlParser->getRootNode();
     if (std::string(root.name()) != nnSkeleton) {
-        printf("\nSkeleton is not a valid root: %s\n", root.name());
+        // printf("\nSkeleton is not a valid root: %s\n", root.name());
```

Comment:
Commenting it out is not removing it. And this really should be a log message.

Reply:
I am not familiar with the context of this log message, so leaving it as a comment for the next person who works on this seemed like the most reasonable thing to do.

If you happen to be familiar with it, please let me know whether I should remove it altogether, or which logging function/macro to use. Thanks!